source/extensions/Sekai.Interop.Common/TimerHelper.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) The Vignette Authors
+// Licensed under MIT. See LICENSE for details.
+
+// Contains code from MonoGame.
+// Copyright 2021 the MonoGame Team.
+// Licensed under the Microsoft Public License and MIT.
+// See https://github.com/MonoGame/MonoGame/blob/develop/LICENSE.txt.
+using System.Runtime.InteropServices;
+
+namespace Sekai.Interop.Common;
+internal static class TimerHelper
```

Comment:
[System.Timers.Timer](https://docs.microsoft.com/en-us/dotnet/api/system.timers.timer?view=net-6.0) is already high resolution, and if you need more look at OTD's native timer implementation. [here](https://github.com/OpenTabletDriver/OpenTabletDriver/tree/master/OpenTabletDriver.Desktop/Interop/Timer) and [here for macos](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/80b98ad170a809399b89aba48387f95d745f3b01/OpenTabletDriver.Desktop/Interop/Timer/MacOSTimer.cs)

Reply:
Removed on my local copy, will push that.